pkg/virt-config/feature-gates.go

```diff
@@ -48,6 +48,7 @@ const (
 	NonRoot                    = "NonRoot"
 	ClusterProfiler            = "ClusterProfiler"
 	WorkloadEncryptionSEV      = "WorkloadEncryptionSEV"
+	DockerSELinuxMCSWorkaround = "DockerSELinuxMCSWorkaround"
```

Comment:
I would suggest leaving a small comment that we basically set `Level` so we don't get `Categories`.

Reply:
Right, enabling the workaround means forcing s0 on non-compute virt-launcher containers. I'll add a comment to that effect.